react-focus: Updating README to reflect package purpose (#12427)
microsoft/fluentui: packages/react-focus/README.md

# @fluentui/react-focus

> **Components that deal with focus management for [Fluent UI](https://fluent.microsoft.com/)**

Reviewer: Wording suggestion plus name and link fix--we should link to our site, not the fluent design system site
```suggestion
**Focus management components for [Fluent UI React](https://dev.microsoft.com/fabric)**
```

> This package contains the different components that deal with focus management to be leveraged when building applications using Fluent UI.

Reviewer: ```suggestion
This package contains various components that deal with focus management, to be leveraged when building applications using Fluent UI.
```

> At the moment the list of these components include:

Reviewer: ```suggestion
At the moment, these components include:
```
